docs/book_files/csi_block_storage_kc_pdfs.md

```diff
@@ -0,0 +1,40 @@
+# Publications and related information
```

Comment:
the `book_files` has 33 files - could we divide it into more dirs some how?
I'm not familiar with the purpose of each file here, but I'm sure we can find some way.
maybe put all the `.dita` files in an inner dir? they are not useful for GitHub users anyway

Reply:
no. the book_files aren't specifically needed for the users in any case. this is the setup that I need for the IBM Docs and PDF content.

Reply:
then we should consider not to merge this dir